### PR TITLE
Removed add button and moved functionality to drop down and enter key.

### PR DESCRIPTION
**Pull request by bledsoef**
Fixes issue #516 

Removed the add button from the user management portion of program settings and allowed the user dropdown to be clickable or for enter to submit.

### CODE REVIEW

File: app/controllers/admin/userManagement.py [hunk 1]
```diff
@@ -16,14 +16,20 @@ def manageUsers():
...
     method = eventData['method']
     username = re.sub("[()]","", (user.split())[-1])
-    user = User.get_by_id(username)
+
+    try:
+        user = User.get_by_id(username)
+    except Exception as e:
+        print(e)
+        flash(username + " is an invalid user.", "danger")
+        return ("danger")
```
Comment (BrianRamsay) [inline app/controllers/admin/userManagement.py]: If it's an error, return an error code. Like `return ("danger"), 500`

File: app/static/js/userManagement.js [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
-    ['removeCeltsAdminInput',       'removeCeltsAdmin',       'admin'],
-    ['removeCeltsStudentStaffInput','removeCeltsStudentStaff','studentstaff']
-  ];
-  $.each(searchElements, function(i,arr) {
-      let [inputId, btnId, category] = arr
-      $("#"+inputId).on("input", () => searchUser(inputId, callback, false, null, category))
-      $("#"+btnId).on("click", () => submitRequest(btnId, $("#"+inputId).val()))
-  });
+  $("#searchCeltsAdminInput").on("input", () => searchUser("searchCeltsAdminInput", callbackAdmin, false, null, 'instructor'))
+  $("#searchCeltsStudentStaffInput").on("input", () => searchUser("searchCeltsStudentStaffInput", callbackStudentStaff, false, null, "student"))
```
Comment (BrianRamsay) [inline app/static/js/userManagement.js]: Let's reformat these to match the other event handlers, now that they are the same as the others

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/controllers/admin/userManagement.py (hunk 1 above)
Comment (Boorclark) [reply app/controllers/admin/userManagement.py]: resolved

File: app/static/js/userManagement.js (hunk 2 above)
Comment (Boorclark) [reply app/static/js/userManagement.js]: resolved

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**